packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -194,6 +194,9 @@ export class EditableText extends AbstractComponent<IEditableTextProps, IEditabl
         if (nextProps.disabled || (nextProps.disabled == null && this.props.disabled)) {
             state.isEditing = false;
         }
+        if (nextProps.value !== this.props.value) {
+            state.lastValue = state.value;
```

Comment:
this means that `props.value` can no longer be updated `onChange`, and should only be updated `onConfirm`.

Is there a reason we expose an `onChange` function?